lib/src/transaction_builder.dart

```diff
@@ -49,13 +47,10 @@ class TransactionBuilder {
           Input(
               sequence: txIn.sequence,
               script: txIn.script,
-              witness: txIn.witness
-          )
-      );
+              witness: txIn.witness));
```

Comment:
Also here, having the parenthesis on a new line at a lower indentation makes it easier to read, as it follows the same principle as braces.

Not sure why this comes up in the lints? When I run them, I get nothing.

Reply:
I have auto format turned on everywhere. 
https://docs.flutter.dev/development/tools/formatting

Dart is very opinionated on this.